packages/zosjobs/src/GetJobs.ts

```diff
@@ -360,4 +381,19 @@ export class GetJobs {
         Logger.getAppLogger().info("GetJobs.getSpoolContentCommon() parameters: " + parameters);
         return ZosmfRestClient.getExpectString(session, JobsConstants.RESOURCE + parameters, [Headers.TEXT_PLAIN_UTF8]);
     }
+
+    private static filterResultsByStatuses(jobs: IJob[], params: IGetJobsParms): IJob[] {
+        if (params?.status && params.status.toLowerCase() != 'active') {
+            const statusJobs = jobs.filter(job => {
```

Comment:
Should this filer operation be simplified to reduce cognitive complexity?
Suggestion: `return jobs.filter(job => job.status.toLowerCase() === params.status.toLowerCase())`

Reply:
Done